bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -130,13 +130,13 @@ export class UIEvents
         if view?
           if view.model.cursor?
             cursor = view.model.cursor()
-          if active_inspectors?
```

Comment:
This name is deceptive, as it indicates we're dealing with a list of objects and not a boolean.